docs/changelog.md

```diff
@@ -57,7 +57,7 @@ _Release date: August 12, 2020_
 
 **Highlights:**
 
-- ⚙️ Ability to set page title, favicon, sidebar state, and wide mode via st.beta_set_page_config(). See our [documentation](https://docs.streamlit.io/en/latest/api.html#streamlit.beta_set_page_config) for details.
```

Comment:
Hmmmm. Not a big deal, but we should probably not change this for historical purposes and make sure the update is added in the next release. @karriebear agree?

Reply:
```suggestion
- ⚙️ Ability to set page title, favicon, sidebar state, and wide mode via st.beta_set_page_config(). See our [documentation](https://docs.streamlit.io/en/0.65.0/api.html#streamlit.beta_set_page_config) for details.
```

This is a good point. Going forward, for change log links to the doc, instead of using latest, we should use the docs for the specific version

Reply:
- Changed link to 0.65.0 in this PR
- Agree that we should try not to change our changelog, but also think that clicking on a link that doesn't work in the changelog would be a worse user experience
- Agreed that removal of beta_ should be called out as an API change, like we do normally!